data/pkgconfig/qt5-boostable.pc

```diff
@@ -1,5 +1,5 @@
 Name: qt5-boostable
 Description: make application boostable by applauncherd
-Version: 0.2.0
+Version: 1.1.17
```

Comment:
This change should be in a separate commit.

Reply:
The ECMGeneratePkgConfigFile will do that automatically it self.

Reply:
If you use that you can remove the pc file already.